src/Stache/Query/Builder.php

```diff
@@ -141,4 +141,37 @@ protected function filterWhereNotIn($values, $where)
             return ! in_array($value, $where['values']);
         });
     }
+
+    protected function filterWhereJsonContains($values, $where)
+    {
+        return $values->filter(function ($value) use ($where) {
+            if (! is_array($value)) {
```

Comment:
Would it make sense to use this for array wrapping instead of returning `false`? And the same with the other method
```php
if (! is_array($value)) {
    $value = [$value];
}
```

Reply:
I dont think so - personally I would want to discourage people from using it on non-json fields as thats what `where()` is for

Reply:
I would check what happens in Laravel when you do whereJsonContains on a string column.

Reply:
They seem to let the database handle it (unless I'm missing something) https://github.com/laravel/framework/blob/03a66a480d5c04ca4911d1cbb72636f4e551bb9b/src/Illuminate/Database/Query/Builder.php#L1669